### PR TITLE
Restyled the admin alerts

### PR DESCRIPTION
**Pull request by hammerfunk**
They also close on click

### CODE REVIEW

File: app/assets/stylesheets/admin/base.scss [hunk 1]
```diff
@@ -174,6 +174,33 @@
...
+  position: fixed;
+  width: 100%;
+  margin: 0 auto;
+  padding: 0 15px;
+  cursor: pointer;
+  z-index: 10001;
+
+  @include screen-sm-min {
+    max-width: 750px;
+  }
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: @include declaration should be followed by an empty line

File: app/assets/stylesheets/admin/base.scss [hunk 2]
```diff
@@ -174,6 +174,33 @@
...
+  @include screen-md-min {
+    max-width: 970px;
+  }
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: @include declaration should be followed by an empty line

File: app/assets/stylesheets/admin/base.scss [hunk 3]
```diff
@@ -174,6 +174,33 @@
   }
 }
 
+// Alerts
+.alert-container {
+  top: 55px;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Properties should be ordered position, top, right, left, width, margin, padding, cursor, z-index

File: app/assets/stylesheets/admin/base.scss (hunk 1 above)
Comment (hammerfunk) [inline app/assets/stylesheets/admin/base.scss]: It does... :unamused:

**Comment by mauricioac** [comment]
Looks good, just waiting for travis

**Pull request merged**